app/src/components/v-form/v-form.vue

```diff
@@ -178,6 +186,24 @@ export default defineComponent({
 
 		const { formFields, getFieldsForGroup, fieldsForGroup, isDisabled, fieldsMeta } = useForm();
 		const { toggleBatchField, batchActiveFields } = useBatch();
+		const { toggleRawField, rawActiveFields } = useRawEditor();
+
+		watch(
+			fields,
+			() => {
+				// make sure to toggle existing values with mustache tags as rawActiveFields
+				// to prevent the initial incompatible values from breaking the original interfaces
+				// on first load
+				for (const formField of formFields.value) {
+					const fieldValue = values.value?.[formField.field] ?? props.initialValues?.[formField.field];
+					if (!fieldValue || typeof fieldValue !== 'string') continue;
+					if (fieldValue.includes('{{') && fieldValue.includes('}}')) {
+						rawActiveFields.value.add(formField.field);
+					}
+				}
+			},
+			{ immediate: true }
+		);
```

Comment:
Instead of auto-switching to raw, we should make sure that the individual interfaces don't crash on unexpected values. I think we can remove this block in favor of tweaking the other interfaces to show a notice if the value can't be edited (eg when `tags` doesn't get an array etc)

Reply:
It's been a hot minute, but I think another reason I did this was to also make it so interfaces such as tags/toggles (and technically somewhat relevant for dropdowns) to be "clearer" that they are using mustache values 🤔

Here's how the tags and toggle fields will look like initially when we edit an operation:

https://user-images.githubusercontent.com/42867097/179159248-32e73fcf-2a26-4b7c-963e-71721a4c55bb.mp4

The crashes should be fixed! This is more so a question of visual/UX I suppose. And I'm not 100% sure what is the intended notice should look like or how can we pivot that for this use case 😅